Jenkinsfile.open.si

```diff
@@ -37,7 +37,8 @@ node('py36') {
                 tokenCredentialId: "4a6636e5-452e-41d2-aea3-9f28448196f3")
         } finally {
             junit allowEmptyResults: true, testResults: "**/shakedown.xml"
-            archiveArtifacts "diagnostics.zip"
+            archive includes: "**/diagnostics.zip"
```

Comment:
From my observations, i have found `archive` step to be not working. See my comment https://github.com/dcos/metronome/pull/341#discussion_r294846776 where it worked after changing `archive` (which is deprecated) to `archiveArtifacts`.

You can allow "empty" archives by adding `, allowEmptyArchive: true` [like here](https://github.com/dcos/metronome/pull/341/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R30)

Either way, if you know that `archive` would work, this lgtm.